tilecloud_chain/server.py

```diff
@@ -498,7 +498,7 @@ def __call__(self):
 
 
 def main(_, **settings):
-    config = Configurator(settings=settings, route_prefix='/tiles')
+    config = Configurator(settings=settings)
```

Comment:
But we need it sometime. Can you make it configurable?
Something like `config = Configurator(settings=settings, route_prefix=os.environ.get('TCC_ROUTE_PREFIX'))`